Library/Homebrew/install.rb

```diff
@@ -40,11 +38,11 @@ def attempt_directory_creation
     end
 
     def check_cc_argv
-      return unless Homebrew.args.cc
+      return unless (cc = args.cc)
```

Comment:
```suggestion
      cc = args.cc
      return unless cc
```

Reply:
I think this pattern is quite idiomatic Ruby.

Reply:
> I think this pattern is quite idiomatic Ruby.

You’re right, it’s [used quite widely](https://github.com/rubocop-hq/ruby-style-guide/tree/0fe5678a01618ce87842b4868363419cb89183d9#safe-assignment-in-condition) but there are people who rightfully oppose it as well ([example](https://gitlab.com/gitlab-org/gitlab-foss/-/issues/45605)).

I think it’s clever but needlessly confusing and [difficult to understand](https://github.com/Homebrew/brew/pull/8084#discussion_r460579310).

Reply:
Agreed with @claui but not a blocker.

Reply:
This check isn't needed anymore now.